src/com/capgemini/service/ReservationService.java

```diff
@@ -67,5 +68,48 @@ public ReservationService() {
                 allReservedRooms.addAll(reservation.getRooms());
         }
         return allReservedRooms;
+
+    }
+
+
+//    private void setSoftDelete(String lastName){
+//
+//        //All the rooms will be stored in this variable
+//        List<Room> lastnameResult = new ArrayList();
+//
+//        //For each reservation in the repository, do the following...
+//        for(Reservation reservation : reservationRepository.getAllReservations()){
+//
+//            //store the last name of the reservation into this variable
+//            String reservationLastName = reservation.getGuest().getLastName();
+//
+//            //if last name equals the reservations last name
+//            if(lastName.equals(reservationLastName)){
+//                if(reservation.isDeleted()){
+//                    System.out.println("This reservation is already deleted");
+//                } else {
+//                    lastnameResult.addAll(reservation.getRooms());
+//                }
+//            } else {
+//                System.out.println("No reservations found");
+//            }
+//        }
+//    }
+
+    private void setSoftDelete(int reservationID){
```

Comment:
is geen setter in principe.. dus ik zou hem "delete" of eventueel "softDelete" noemen